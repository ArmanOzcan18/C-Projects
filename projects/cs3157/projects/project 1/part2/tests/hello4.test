-f words/short-list -n 0 -g 3 play

