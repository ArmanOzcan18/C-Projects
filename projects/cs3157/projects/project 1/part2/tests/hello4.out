Guess:_-_+_
You correctly guessed 2 characters.
Guess:_____
You correctly guessed 0 characters.
Guess:--+--
You correctly guessed 5 characters.
Guess:Game aborted. The answer was: hello
